ddtrace/tracer/option.go

```diff
@@ -215,6 +218,14 @@ func WithDogstatsdAddress(addr string) StartOption {
 	}
 }
 
+// WithSamplingRules specifies the sampling rates to apply to spans based on the
+// provided rules.
+func WithSamplingRules(rules ...SamplingRule) StartOption {
```

Comment:
Can we please have a slice here as we’ve discussed? The definition becomes easier because you don’t have to keep repeating the type on every rule.